src/README.md

```diff
@@ -1,122 +1,30 @@
-
-Repository <br>
-```
-git@github.com:ripple/LevelDB.git
-```
-Branch
-```
-ripple-fork
-```
-
-## LightningDB (a.k.a. MDB)
-
-Ripple's fork of MDB, a fast memory-mapped key value database system.
-
-Repository <br>
-```
-git@github.com:ripple/LightningDB.git
-```
-Branch
-```
-ripple-fork
-```
-
-## websocket
-
-Ripple's fork of websocketpp has some incompatible changes and Ripple specific includes.
-
-Repository
-```
-git@github.com:ripple/websocketpp.git
-```
-Branch
-```
-ripple-fork
-```
-
-## protobuf
-
-Ripple's fork of protobuf. We've changed some names in order to support the
-unity-style of build (a single .cpp added to the project, instead of
-linking to a separately built static library).
-
-Repository
-```
-git@github.com:ripple/protobuf.git
-```
-Branch
-```
-master
-```
-
-**NOTE** Linux builds use the protobuf installed in /usr/lib. This will be
-fixed in a future revision.
-
-## SQLite
-
-Not technically a subtree but included here because it is a direct
-copy of the official SQLite distributions available here:
-
-http://sqlite.org/download.html
+[git-subtree]: https://github.com/apenwarr/git-subtree
+
+If you create a commit that contains files both from a subtree, and from the
+`rippled` source tree, please use care when designing the commit message, since
+it will appear in the subtree's individual repository when the changes are
+pushed back to the upstream. Better yet, do not mix files from subtrees and
+`ripple` in the same commit at all.
+
+Source folders:
+
+| Folder          | Upstream Repo                                | Description |
+|:----------------|:---------------------------------------------|:------------|
+| `beast`         | https://github.com/vinniefalco/Beast         | Cross-platform library for WebSocket and HTTP built on [Boost.Asio](https://think-async.com/Asio) |
+| `ed25519-donna` | https://github.com/floodyberry/ed25519-donna | [Ed25519](http://ed25519.cr.yp.to/) digital signatures |
+| `lz4`           | https://github.com/lz4/lz4                   | LZ4 lossless compression algorithm |
+| `nudb`          | https://github.com/vinniefalco/NuDB          | Constant-time insert-only key/value database for SSD drives (Less memory usage than RocksDB.) |
+| `protobuf`      | https://github.com/google/protobuf           | Protocol buffer data interchange format. Ripple has changed some names in order to support the unity-style of build (a single .cpp added to the project, instead of linking to a separately built static library). |
+| `ripple`        | N/A                                          | **Core source code for `rippled`** |
+| `rocksdb2`      | https://github.com/facebook/rocksdb          | Fast key/value database. (Supports rotational disks better than NuDB.) |
+| `secp256k1`     | https://github.com/bitcoin-core/secp256k1    | ECDSA digital signatures using the **secp256k1** curve |
+| `snappy`        | https://github.com/google/snappy             | "Snappy" lossless compression algorithm. |
```

Comment:
it looks like it's actually `snappy/snappy` that is the subtree (very nitpicky), and the `config` dir we created probably to avoid needing to run `configure` on the subtree first (that's just a guess). In any event, I'm not sure this level of detail is required.